firmware/integration/rusefi_config.txt

```diff
@@ -861,7 +861,18 @@ spi_device_e accelerometerSpiDevice;
 	float[FSIO_CURVE_8] fsioCurve3;;"y",      1,     0,      -999,    1000.0,   3
 	float[FSIO_CURVE_8] fsioCurve4Bins;;"x",        1,     0,   -999,    1000.0,  3
 	float[FSIO_CURVE_8] fsioCurve4;;"y",      1,     0,      -999,    1000.0,   3
-	int[794] unusedEnd;
+
+	bit useFixedBaroCorrFromMap
```

Comment:
We had a few unused bits within existing bit maps - see commit https://github.com/rusefi/rusefi/commit/2c4c8771bb3fdc0e2ce89d03441eafbb61eec15a

please merge - I've already added "useFixedBaroCorrFromMap" into my commit